### PR TITLE
r1.9: release notes Update.

### PR DESCRIPTION
**Pull request by MarkDaoust**
I haven't submitted this to master... should I?

- link to new get_started.
- Add keras CuDNN layers.
- Links for gradient boosted estimators.
- Added new contrib-estimators and string-processing.
- Bumped some minor sounding things down from "Major" to "Bugfix+Other"

### CODE REVIEW

**Review by lamberta — APPROVED** [review]

**Review by case540 — APPROVED** [review]

**Comment by case540** [comment]
Would appreciate all changes to be submitted to master and then cherry-picked over to release branches to keep things simpler. Please submit these changes to master branch when you get the chance. ty!

**Comment by MarkDaoust** [comment]
>Please submit these changes to master branch when you get the chance.

Done:

https://github.com/tensorflow/tensorflow/pull/20291
